### DISCUSSION TITLE
Daily 05/05

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#212] [Daily Sprint]
- Qué hice ayer?
- Qué haré hoy?
- Dificultades e impedimentos

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve haciendo las modificaciones en la barra de navegación, en el footer y en la página de contacto. Por la tarde tuve clase de programación. No llegue a hacer el ejercicio de Inglés.
Hoy realicé la tarea de Inglés. Más tarde estaré haciendo las modificación de pasar el dashboard al footer. También estaré revisando la documentación para ver que hay que modificar o agregar con respecto a este sprint. Por la noche nos volveremos a juntar para evaluar cómo estamos.

**Comment by rpgrca** [comment]
Hoy tuve que ir a Capital así que a la mañana no hice mucho. Ahora estoy haciendo el ABM para los tipos de productos para ir terminando con el dashboard.

**Comment by BraianTroncoso** [comment]
Ayer modifiqué el logo, le cambié el color y lo pasé a svg. También me puse con java y un poco de python. Hoy vi un poco de Angular sobre servicios.
